modules/core/src/main/java/org/apache/ignite/internal/management/api/WithCliConfirmParameter.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.management.api;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Marks commands that must output example of optional {@code --yes} parameter.
+ * Only for compatibility with existing {@code ./control.sh} output.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface WithCliConfirmParameter {
```

Comment:
Parameter --> Argument? Why we annotate command argument class? Let's mark commands instead (as these docs actually says).

I suppose, in general argument classes can be reused? For example NoArg.class.

Reply:
> Why we annotate command argument class?

Because command arguments and description derived from argument class.
It seems inconvinient to pass command class of flag only to check this single annotation.